contracts/modules/Payments.sol

```diff
@@ -61,12 +61,16 @@ abstract contract Payments is RouterImmutables {
         if (token == Constants.ETH) {
             uint256 balance = address(this).balance;
             uint256 amount = (balance * bips) / FEE_BIPS_BASE;
-            recipient.safeTransferETH(amount);
+            uint256 fixedAmount = (amount * PAYMENT_AMOUNT_BIPS) / FEE_BIPS_BASE;
```

Comment:
Our past routers have had the option to let third parties use this feature too, are we ok with getting rid of that?

Reply:
I'm confused... what about this doesnt let a third party use it?
The design is:
- x% is taken from the user
- `PAYMENT_AMOUNT_BIPS`% of the x% goes to the recipient
- the rest of the x% goes to the address specified by the 3rd party